docs/9-Github/9.2-Actions.md

```diff
@@ -0,0 +1,88 @@
+---
+docs/9-Github/9.2-Actions.md:
+  category: Fundamentals
```

Comment:
```suggestion
  category: CI/CD
```

This _feels_ like a better category for GHA. If you are wondering where I am pulling these you can see existing categories if you scroll down on the home page of the bootdcamp. There is a doughnut chart midway down the page